vars/cloudFoundryServiceCreate.groovy

```diff
@@ -0,0 +1,176 @@
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.BashUtils
+import com.sap.piper.JenkinsUtils
+import com.sap.piper.Utils
+import com.sap.piper.ConfigurationHelper
+
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = 'cloudFoundryServiceCreate'
```

Comment:
according to the the naming pattern we use, the step name should rather be `cloudFoundryCreateService`

Default configuration should then also match this name.

Reply:
Was quite unsure how to name the step. Will change it.